apis/gateway/v1alpha1/filter_types.go

```diff
@@ -0,0 +1,47 @@
+// Package v1alpha1 contains custom resource definitions for use with the Gateway API.
+//
+//nolint:lll
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// PolicyFilter represents a Pomerium policy that can be attached to a particular route defined
+// via the Kubernetes Gateway API.
+//
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
+type PolicyFilter struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// Spec defines the content of the policy.
+	Spec PolicyFilterSpec `json:"spec,omitempty"`
+
+	// +kubebuilder:default={conditions: {{type: "Valid", status: "Unknown", reason:"Pending", message:"Waiting for controller", lastTransitionTime: "1970-01-01T00:00:00Z"}}}
```

Comment:
This is unusual, but I suppose it should work. Is this a gateway API requirement?

Reply:
It's not a requirement, but I wanted to be as consistent as possible with existing Gateway API conventions. I based this on how the Gateway [Status](https://github.com/kubernetes-sigs/gateway-api/blob/1fe2b9f8ee99a6475a65eedd1ce060f363a8634d/apis/v1/gateway_types.go#L44) and [Conditions](https://github.com/kubernetes-sigs/gateway-api/blob/1fe2b9f8ee99a6475a65eedd1ce060f363a8634d/apis/v1/gateway_types.go#L692) fields were annotated.

Is there a more conventional way to use status conditions?

Reply:
Oh, I see. I think that is gateway api-specific; their resources are designed to not have a controller by default, and become controlled by one after creation, so the default status makes sense for them. We probably don't need to do the same here.

Reply:
To clarify, keep status and conditions as-is, but you can remove the kubebuilder default option.